Scripts/Spice.py

```diff
@@ -17,14 +18,188 @@ def compute_view_frustrum(instrument):
 
 def spacecraft_position_and_orientation(spacecraft, point):
     """
-    stub
+    Look under SPK Kernel for now
     :param spacecraft: Data structure that models the spacecraft.
     :param point: point in time.
     :return:
     """
     pass
 
-
 """
 Create Spice Wrappers Here:
-"""
\ No newline at end of file
+"""
+
+###########
+# Kernels #
+###########
+
+class Kernel(str):
```

Comment:
Instead of a class inheriting from `str`, this should either inherit from [`Enum`](
https://docs.python.org/3/library/enum.html) or just be a set literal. The latter is probably the easiest way to go if all we want is to determine if a kernel type is valid.

Reply:
https://github.com/kylrbutlr/CraterCapstone/blob/6a65b69bac6a288f1cd591719e21da8e22d2fa1f/Scripts/Spice.py#L37

Woops was confused how enums worked in python should be fixed now